.github/workflows/dev_on_push_update_jira_issues.yml

```
@@ -0,0 +1,152 @@
+          fetch-depth: 0 # Fetch all history for "git rev-list --count HEAD" info
+
+      - name: Setup gajira cli
+        uses: atlassian/gajira-cli@master
+        with:
+          version: 1.0.20
+
+      - name: Authentification to Google
+        uses: 'google-github-actions/auth@v1'
+        with:
+          workload_identity_provider: ${{ secrets.GCP_EHP_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.GCP_EHP_SERVICE_ACCOUNT }}
+
+      - name: Get Secret
+        id: 'secrets'
+        uses: 'google-github-actions/get-secretmanager-secrets@v1'
+        with:
+          secrets: |-
+            JIRA_USER_EMAIL:passculture-metier-ehp/passculture-app-native-jira-user-email
+            JIRA_API_TOKEN:passculture-metier-ehp/passculture-app-native-jira-api-token
+
+      - name: Login
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: https://passculture.atlassian.net/
+          JIRA_USER_EMAIL: ${{ steps.secrets.outputs.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ steps.secrets.outputs.JIRA_API_TOKEN }}
+
+      - name: Get jira previous commit number
+        id: get_previous_commit_number
+        env:
+          JIRA_API_TOKEN: ${{ steps.secrets.outputs.JIRA_API_TOKEN }}
+        # The jira view command uses .jira.d/templates/view template by default
+        run: |
+          echo "value=$(jira view ${{ needs.issue-number.outputs.status }})" >> $GITHUB_OUTPUT
+
+      - name: Mark issue as not deployable
+        env:
+          JIRA_API_TOKEN: ${{ steps.secrets.outputs.JIRA_API_TOKEN }}
+        if: ${{ steps.get_previous_commit_number.outputs.value != null && steps.get_previous_commit_number.outputs.value != '<no value>' }}
+        # If previous_commit_number is not null, it means this push-to-master is a fix.
+        # All commits with number between previous_commit_number and this commit number are therefore not safe to deploy.
+        # So mark them as "impropre: true".
+
+        # The echo command is to clear jira config.yml file set in get jira issue step
+        # The jira edit command uses .jira.d/templates/edit template by default
+        run: |
+          echo "issue:" >> $HOME/.jira.d/config.yml
+          jira edit --query='"Numéro de commit[Number]" > ${{ steps.get_previous_commit_number.outputs.value }} AND "Repository[Short text]" ~ "pass-culture-app-native"' --override $IMPROPRE_FIELD=true --noedit
+
+      - name: Push commit number and commit hash to jira issue
+        env:
+          JIRA_API_TOKEN: ${{ steps.secrets.outputs.JIRA_API_TOKEN }}
+        run: |
+          jira edit ${{ needs.issue-number.outputs.status }} \
+            --override $IMPROPRE_FIELD=false \
+            --override $COMMIT_NUMBER_FIELD=${{ needs.issue-number.outputs.commit-number }} \
+            --override $COMMIT_HASH_FIELD=${{ needs.issue-number.outputs.commit-hash }} \
+            --override $REPOSITORY_FIELD="pass-culture-app-native" \
+            --noedit
```

suggestion(noblocking): on pourrait simplifier l'écriture et la lecture https://yaml-multiline.info/

```suggestion
        run: >
          jira edit ${{ needs.issue-number.outputs.status }}
            --override $IMPROPRE_FIELD=false
            --override $COMMIT_NUMBER_FIELD=${{ needs.issue-number.outputs.commit-number }}
            --override $COMMIT_HASH_FIELD=${{ needs.issue-number.outputs.commit-hash }}
            --override $REPOSITORY_FIELD="pass-culture-app-native"
            --noedit
```